docs/modules/user-guide/partials/proc_devfile-quick-reference.adoc

```diff
@@ -0,0 +1,263 @@
+[id="proc_devfile-quick-reference_{context}"]
+= Devfile quick reference
+
+[role="_abstract"]
+This page quickly introduces you to devfile properties. The code snippets help you get started developing your own devfile projects. The tables show you what the different devfile properties require if you use these properties. Refer to the table of contents to go to the devfile property you want to work with.
+
+== SchemaVersion
```

Comment:
It looks like schemaVersion is a proper name, perhaps it should begin with a lowercase letter even in a title, WDYT? E.g. eBay wouldn't be spelled EBay in a title. But I haven't managed to find a relevant rule about that in the IBM SG.

Reply:
You're right. Proper names and nouns are always spelt the same way no matter where they are in headings or sentences.